src/app/routes/stake/stake-list/stake-list.component.html

```diff
@@ -1,8 +1,5 @@
 <div class="stake-list">
     <h4 class="color-white">Providers</h4>
-
-    <img src="/assets/img/icons/stake-list-banner.svg" alt="stake-list-banner" class="stake-list-banner">
```

Comment:
no banner?

Reply:
Hide for now. Currently doing a unique banner.